### ISSUE TITLE
Extend README and make it newbie-friendly 

### ISSUE THREAD

**Issue by DanStockham** [FCCColumbus/F3C_Website#18]
Right now, the current documentation helps when setting up the project and environment. However, newer users may still get lost after setting up the environment and start coding. Therefore, we should explain more after the user sets up the project. Include a file structure and highlight directories the user would be working mostly in.

Just some thoughts...

**Comment by jwoo92** [comment]
To add to this thought, it might not be a bad idea to have a small section that helps others setup linters on their preferred choice of text editor (e.g. Sublime, Atom, etc.). We have it built into the project, but it is nice to have in the text editor as well, which unfortunately is not an out of the box feature for most.

I also have a few sublime snippets I can share if you guys think it would be helpful.

List of sublime snippets:

- React statefull class component (container component)
- React stateless functional component (presentational component)
- React Redux Connect component
- React Redux Connect add-on snippet for converting from an existing class component to Redux/Connect

**Comment by Kavinci** [comment]
Per slack discussion and my thoughts:

- Establish workflow
- Who to contact for questions, long waits on PR approval, etc.
- List of technologies used in code and deployment

**Comment by Kavinci** [comment]
After reading these again and thinking this could also be written as a wiki as well.
